File: src/sendMessage.ts
import {Twilio} from "twilio";
import env from "dotenv";
env.config()

const accountSid = process.env.TWILIO_ACCOUNT_SID || "";
const authToken = process.env.TWILIO_AUTH_TOKEN || "";
const fromPhone = process.env.FROM_PHONE_NUMBER || "";
const zacPhone = process.env.ZAC_PHONE_NUMBER || "";

const client = new Twilio(accountSid, authToken);

const sendText = (body:string) => {
	client.messages
	.create({
		body: body,
		from: fromPhone,
		to: zacPhone
	})
	.then(message => console.log(message.sid));
}

export {sendText}

File: server/send_message.py
from twilio.rest import Client
from dotenv import dotenv_values

def send_text(message):
	env = dotenv_values(".env")  
	account_sid = env['TWILIO_ACCOUNT_SID']
	auth_token = env['TWILIO_AUTH_TOKEN']
	client = Client(account_sid, auth_token)
	message = client.messages \
					.create(
						body=message,
						from_=env['FROM_PHONE_NUMBER'],
						to=env['ZAC_PHONE_NUMBER']
					)
